crates/transaction-pool/src/config.rs

```diff
@@ -107,6 +108,15 @@ impl SubPoolLimit {
     }
 }
 
+impl Mul<usize> for SubPoolLimit {
+    type Output = Self;
+
+    fn mul(self, rhs: usize) -> Self::Output {
+        let Self { max_txs, max_size } = self;
+        Self { max_txs: max_txs * 2, max_size: max_size * rhs }
```

Comment:
was it meant to be this?
```suggestion
        Self { max_txs: max_txs * rhs, max_size: max_size * rhs }
```

Reply:
ah no lol